src/Prima/Pyxis/DownloadButton.elm

```diff
@@ -0,0 +1,416 @@
+-}
+withOnMouseOut : msg -> Config msg -> Config msg
+withOnMouseOut tagger =
+    addOption (Event (Events.onMouseOut tagger))
+
+
+{-| Adds a generic attribute to the Button.
+-}
+withAttribute : Html.Attribute msg -> Config msg -> Config msg
+withAttribute attr =
+    addOption (Attribute attr)
+
+
+{-| Renders the button by receiving it's configuration.
+
+    --
+
+    import Prima.Pyxis.DownloadButton as DownloadButtonButton
+
+    type Msg =
+        Clicked
+
+    ...
+
+    myBtn : DownloadButton.Config Msg
+    myBtn =
+        DownloadButton.default "Certificato di polizza" "Scarica .pdf"
+            |> DownloadButton.withDisabled False
+            |> DownloadButton.withOnClick Clicked
+
+    ...
+
+    view : Html Msg
+    view =
+        DownloadButton.render myBtn
+
+-}
+render : Config msg -> Html msg
+render ((Config { title, subtitle }) as config) =
+    button
+        (buildAttributes config)
+        [ i
+            [ class "a-btn--download__icon a-icon a-icon-download" ]
+            []
+        , div
+            [ class "a-btn--download__title" ]
+            [ text title ]
+        , div
+              [ class "a-btn--download__subtitle" ]
+              [ text title ]
+        ]
+
+
+{-| Internal. Transforms all the customizations into a list of valid Html.Attribute(s).
+-}
+buildAttributes : Config msg -> List (Html.Attribute msg)
+buildAttributes buttonConfig =
+    let
+        options =
+            computeOptions buttonConfig
+    in
+    [ options.id
+        |> Maybe.map Attrs.id
+    , options.tabIndex
+        |> Maybe.map Attrs.tabindex
+    , options.disabled
+        |> Maybe.map Attrs.disabled
+    ]
+        |> List.filterMap identity
+        |> (::) (buildType options)
+        |> (::) (buildFormTarget options)
+        |> (++) (buildClasses options)
+        |> List.append options.attributes
+        |> List.append options.events
+
+
+{-| Internal. Constructs the `type` attribute from the `DownloadButton` options.
+-}
+buildType : Options msg -> Html.Attribute msg
+buildType options =
+    case options.type_ of
+        Button ->
+            Attrs.type_ "button"
+
+        Submit ->
+            Attrs.type_ "submit"
+
+        Reset ->
+            Attrs.type_ "reset"
+
+
+{-| Internal. Constructs the `formtarget` attribute from the `DownloadButton` options.
+-}
+buildFormTarget : Options msg -> Html.Attribute msg
+buildFormTarget options =
+    case options.target of
+        Self ->
+            Attrs.attribute "formtarget" "_self"
+
+        Blank ->
+            Attrs.attribute "formtarget" "_blank"
+
+        Parent ->
+            Attrs.attribute "formtarget" "_parent"
+
+        Top ->
+            Attrs.attribute "formtarget" "_top"
+
+
+{-| Internal. Merges the component configuration and options to a classes attribute.
+-}
+buildClasses : Options msg -> List (Html.Attribute msg)
+buildClasses options =
+    [ Attrs.class "a-btn a-btn--download"
```

Comment:
Avendo definito le default classes direttamente nel record `defaultOptions` qui puoi evitare questa riga e trasformare la firma di questa funzione in `buildClasses : Options msg -> Html.Attribute msg`